addons/payment/models/payment_acquirer.py

```diff
@@ -800,6 +800,7 @@ def _cron_post_process_after_done(self):
                 self.env.cr.commit()
             except Exception as e:
                 _logger.error("Transaction post processing failed, reason \"%s\"", e)
```

Comment:
Wouldn't it be easier to debug (and fix problems) if we log the traceback?
```suggestion
                _logger.exception("Transaction post processing failed")
```

Reply:
Done :)